fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/database/hibernate/PrefixAndSnakeCasePhysicalNamingStrategy.java

```diff
@@ -0,0 +1,53 @@
+package de.adorsys.opba.fintech.impl.database.hibernate;
+
+import com.google.common.base.CaseFormat;
+import org.hibernate.boot.model.naming.Identifier;
+import org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl;
+import org.hibernate.engine.jdbc.env.spi.JdbcEnvironment;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Converts Hibernate snakeCase naming into Database-friendly camel_case, also adds prefix to the tables.
```

Comment:
Change to
`Converts Hibernate camelCase naming into Database-friendly snake_case, also adds prefix to the tables.`